404.html

```diff
@@ -1,6 +1,13 @@
 ---
 ---
-
+  <script type="text/javascript">
```

Comment:
Can't we instead rely on some query param that would be put on the URL by the documentation redirect? This would much reduce the scope of this automatic redirect.
Something like `http://ehcache.org/documentation/3.1/expiry.html?source=old` and you would redirect to the documentation root only if the query param is present.
Wdyt?